### PR TITLE
Remove excess logging

### PR DESCRIPTION
**Pull request by technoir42**

### CODE REVIEW

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt [hunk 1]
```diff
@@ -83,10 +81,7 @@ class PullScreenshotTestRunListener(
 
     private fun removeScreenshots(applicationId: String) {
         val remoteDir = device.fileManager.getScreenshotsDir(applicationId)
-        val millis = measureTimeMillis {
-            device.fileManager.removeFromFilesDir(applicationId, remoteDir)
-        }
-        logger.trace { "Removed files from $remoteDir in ${millis}ms" }
```
Comment (paulmihaicraciunas) [inline vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt:89]: I had a look at VRT outputs and this is 1 line of log. I doubt it makes much of a difference.

I think a better candidate would be where we write the test name.

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt [hunk 2]
```diff
@@ -64,8 +64,6 @@ class PullScreenshotTestRunListener(
         val remoteDir = device.fileManager.getScreenshotsDir(applicationId)
         val outputDir = outputDirectory.toFile()
 
-        logger.trace { "Pulling screenshots from $remoteDir" }
```
Comment (paulmihaicraciunas) [inline vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt:67]: Again, this is only logged once. Does it make that much of a difference?

File: core/src/main/kotlin/com/malinskiy/marathon/cache/test/TestCacheSaver.kt [hunk 3]
```diff
@@ -26,13 +24,8 @@ class TestCacheSaver(
     fun initialize(scope: CoroutineScope) = with(scope) {
         completableDeferred = async {
             for (task in tasks) {
-                val storingTime = measureTimeMillis {
-                    val cacheKey = testCacheKeyProvider.getCacheKey(task.poolId, task.result.test)
-                    cache.store(cacheKey, task.result)
-                }
-                logger.debug {
-                    "Writing test result to cache for ${task.result.test.toSimpleSafeTestName()} took $storingTime milliseconds"
```
Comment (paulmihaicraciunas) [inline core/src/main/kotlin/com/malinskiy/marathon/cache/test/TestCacheSaver.kt:34]: Is this 1 line really excessive?

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt (hunk 2 above)
Comment (technoir42) [reply vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt:67]: It's logged once per test, and below we already log the dir name after pulling is finished.

File: vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt (hunk 1 above)
Comment (technoir42) [reply vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt:89]: I replaced logging of `Test` with test name where possible.

**Review by paulmihaicraciunas — APPROVED** [review]

**Pull request merged**